src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/BlockPlacer.java

```diff
@@ -146,6 +146,10 @@ private boolean isBlacklisted(Material type) {
     private void placeSlimefunBlock(SlimefunItem sfItem, ItemStack item, Block block, Dispenser dispenser) {
         BlockPlacerPlaceEvent e = new BlockPlacerPlaceEvent(dispenser.getBlock(), item, block);
         Bukkit.getPluginManager().callEvent(e);
+        
+        if(!dispenser.getInventory().getViewers().isEmpty()){
```

Comment:
Formatting, please adhere to our code style guidelines, there should be a space after the `if` and before the curly bracket `{`